allennlp/training/metrics/conll_scores.py

```diff
@@ -0,0 +1,155 @@
+from typing import Tuple
+from collections import Counter
+import numpy as np
+from sklearn.utils.linear_assignment_ import linear_assignment
+
+from overrides import overrides
+
+from allennlp.training.metrics.metric import Metric
+
+@Metric.register("conll_scores")
+class ConllScores(Metric):
```

Comment:
There are lots of CoNLL datasets, each with their own metric.  This should be something like `CorefScores`, or `ConllCorefScores` if you want to keep the reference to the CoNLL coref metrics (as opposed to, e.g., plain b-cubed).

Reply:
Fixed.